### PR TITLE
Add Views

### PR DESCRIPTION
**Pull request by kwesiObuobi**
### In this PR, I did the following:
- Added route-view rendering
- Added views to the project, according to the requirements
- Used the methods I created in the models

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+  end
```
Comment (christianbiring1) [inline app/controllers/posts_controller.rb:9]: - [x] As you may notice, once creating a user, It prompts me to insert the `post_counter` parameter as well which is supposed to have a default of `0` in your database and be incremented as the user is posting on the platform, Kindly make sure you correct that in your `User Models`
image

hint: You may consider saying for example:
```rb
class User < ApplicationRecord
  #has_many :posts, foreign_key: :author_id
  #has_many :comments, foreign_key: :author_id
  #has_many :likes, foreign_key: :author_id

  #validates :name, presence: true
  #validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
  after_initialize :initialize_post-counter
  before_create :set_default_avatar #here you can consider adding a fx that will create a default image avatar when the user is created or you can leave it for now as you have the URL provided for now

  #scope :three_recent_posts, ->(user) { user.posts.order(created_at: :desc).limit(3) }
#end
def initialize_post-counter #This method aims to initialize the number of post to zero once a post is created if the user haven't yet post anything.
    self.posts_count ||= 0
    true
  end
```

Comment (christianbiring1) [reply app/controllers/posts_controller.rb:9]: - [x] For the comment you will need to do the same thing

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @kwesiObuobi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 


## Highlights
- [ ] Great wok work for rendering the view of the as suggested by the wireframe template :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (kwesiObuobi) [reply app/controllers/posts_controller.rb:9]: Big thanks, @christianbiring1

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @kwesiObuobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] All views have been created :white_check_mark: 
- [x] Application works as expected :white_check_mark: 
- [x] Pull request has a descriptive summary and title :white_check_mark: 
- [x] Implemented all the changes from the previous reviewer :white_check_mark:   

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (christianbiring1) [reply app/controllers/posts_controller.rb:9]: you're welcome @kwesiObuobi

**Pull request merged**